src/create-account/AccountInfoForm.js

```diff
@@ -46,12 +46,16 @@ export default class CreateAccountCard extends Component {
 
     proceed() {
         const { actions, email } = this.props;
+
         actions.createAccountFieldUpdate('validatedOnce', true);
         actions.createAccountFieldUpdate('progress', true);
         if (this.emailValid() && this.passwordValid() && this.confirmationValid()) {
             LiveData.api.verifyEmail(email, 'account_opening').then(() => {
                 actions.createAccountFieldUpdate('step', 1);
                 actions.createAccountFieldUpdate('progress', false);
+            })
+            .catch((err) => {
```

Comment:
Wanted to , but on a second thought i didn't considering the light action performed.